src/main/java/roomescape/member/domain/Member.java

```diff
@@ -2,18 +2,33 @@
 
 import java.util.Objects;
 
+import jakarta.persistence.AttributeOverride;
+import jakarta.persistence.Column;
+import jakarta.persistence.Embedded;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
 public class Member {
 
-    private static final long EMPTY_ID = 0L;
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
저는 한 줄에 하나의 애너테이션만 위치시키는게 가독성이 좋다고 생각해서 자주 활용하는편인데요. 
포스티는 한 줄에 여러 애너테이션을 위치시키셨군요? 👀 
애너테이션이 더 추가된다면 다음 애너테이션은 어디에 위치시키실건가요? 🤔 

이 부분은 정해진 방식이 있는 것은 아니지만, 제가 사용하는 방식도 한번 가볍게 생각해봐주시면 좋을 것 같아요. 😃

```suggestion
    @Id 
    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Reply:
클래스, 필드에 붙이는 어노테이션은 한 줄에 하나만 위치시키는 것이 평소 컨벤션인데 이번 미션은 왠지 아무 생각없이 넘어간것 같네요..

로키가 `suggested chage`에 남겨주신 것 처럼 평소에는 위와같은 컨벤션을 지키며 프로그래밍합니다.

여기서 궁금한 점이 하나 있는데, 로키는 메서드 파라미터에 붙이는 어노테이션의 경우에도 한 줄에 하나의 어노테이션을 지정해주시나용?
1번 방법과 2번 방법 중 어떤 방법을 평소에 선호하시나용?

- 1번
```java
@RequiredAdmin
@PostMapping
public ResponseEntity<ReservationResponse> create(
        @Valid @RequestBody final AdminReservationRequest request
) {
    ...
}
```

- 2번
```java
@RequiredAdmin
@PostMapping
public ResponseEntity<ReservationResponse> create(
        @Valid
        @RequestBody
        final AdminReservationRequest request
) {
    ...
}
```

Reply:
메서드 파라미터의 경우에는 1번처럼 쓰고있어요! 
사실 선호한다기보다는 이렇게 사용하는 케이스 밖에 거의 못본터라 이렇게 쓰는 것이 일반적인 것 같아서 저도 코딩스타일이 이렇게 굳혀진 듯한 느낌이네요. 🤔

2번 같은 스타일은 메서드 파라미터가 조금만 많아지면 메서드 라인 수가 너무 커질 우려가 있어서 그런게 아닐까...? 라는 생각이 드네요.